examples/apps/meshdemo/README.md

```diff
@@ -0,0 +1,371 @@
+# Graphic AWS App Mesh Demo
+
+In this demo we will deploy AWS App Mesh with the X-Ray daemon using tools like IAM Service Accounts with kubectl -k (kustomize) to demonstrate some of the AWS App Mesh features.
+
+The tools needed for this demo are:
+- eksctl
+- kubectl (> 1.13)
+
+## Setup
+
+### Deploy a cluster
+Create a cluster using eksctl
+```bash
+eksctl create cluster --name=meshdemo --region=eu-west-1 --nodes 1 --appmesh-access --version 1.14
+```
+
+### OIDC Provider for IAM Service Accounts
+We will associate our IAM OIDC provider, so we can use service accounts. 
+```bash
+eksctl utils associate-iam-oidc-provider --name meshdemo --approve
+```
+
+### Clone
+Clone this repository
+```
+git clone git@github.com:aws/aws-app-mesh-examples.git
+cd aws-app-mesh-examples/examples/apps/meshdemo
+```
+
+Deploy the [AWS App Mesh Controller](https://github.com/aws/aws-app-mesh-controller-for-k8s).
+```bash
+kubectl apply -k kubernetes/mesh/kustomize/mesh-controller
+```
+This also creates a namespace "meshdemo" for the below service account, and creates a mesh called "meshdemo".
+
+## Create a Service Account
+We create a service account so the X-Ray daemon can push to the X-Ray endpoint. 
+```bash
+eksctl create iamserviceaccount --name xray-access --namespace meshdemo --cluster meshdemo \
+--attach-policy-arn arn:aws:iam::aws:policy/AWSXRayDaemonWriteAccess --approve --override-existing-serviceaccounts
+```
+
+## The Demo
+
+The demo consists of a few services designed specifically to demonstrate AWS App Mesh features.
+
+![](images/appmesh-graphic-demo.png)
+
+- Frontend: constantly polls the backends for values.
+- Colors: Blue and green service are both pods running in the namespace currently only blue is being routed
+- Cats: is a service that requests a random cat gif image everytime the frontend polls it.  When it is first started
+it cannot reach the external service.
+
+All the code for the applications can be found in the cat, colors and frontend folders of this folder.
+
+### Running the services
+First we will start the base of the services.  This deploys the services, with an AWS X-Ray sidecar and an App Mesh sidecar.
+
+```bash
+kubectl apply -k kubernetes/app/kustomize/base
+```
+
+To view the frontend we can port-forward the service using kubectl.
+```bash
+k port-forward -n meshdemo svc/frontend 8080:8000
```

Comment:
you may need to add your aliases above :)

Reply:
I think I removed all aliases